sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -128,7 +128,18 @@ Licensed under the MIT License.
         </exclusion>
       </exclusions>
     </dependency>
-
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-tracing-opentelemetry</artifactId>
+      <scope>test</scope>
+      <version>1.0.0-beta.5</version> <!-- {x-version-update;com.azure:azure-core-tracing-opentelemetry;dependency} -->
```

Comment:
What is the timeline for GA of this package - Seems weird if a part of the SDK officially GAed (azure-cosmos) takes a dependency on a component still in Beta.

Reply:
@FabianMeiswinkel Since this is a test dependency  -> `<scope>tests<scope>`.I think we should be fine, right @simplynaveen20  ?

Reply:
That is correct, this is test dependency . Mostly users gonna onboard tracing with GA version version only . And once we have that we can move this test dependency to GA as well . So overall we should be good now,

Reply:
we are now not using this test dependency